cms/templates/widgets/header.html

```diff
@@ -1,4 +1,3 @@
-<%page expression_filter="h"/>
```

Comment:
Reviewers and @nasthagiri: Update: I decided to implement the fixes rather than reverting the issue to help make forward progress.